drud/ddev
# Add pecl uploadprogress to web container, fixes #1249

## The Problem/Issue/Bug:

OP #1249 PECL Uploadprogress. This is always "nice to have" in the Drupal world

## How this PR Solves The Problem:

## Manual Testing Instructions:

You can test this either with the artifacts from CircleCI or just by using

`webimage: drud/ddev-webserver:20181124_pecl_upload_progress` 

in your .ddev/config.yaml

* Start a Drupal 8 project, visit admin/reports/status to see uploadprogress enabled
* Start a Drupal 7 project with *apache*, visit admin/reports/status to see it enabled

## Automated Testing Overview:
None

## Related Issue Link(s):

OP #1249
